download/alpha.html

```diff
@@ -11,17 +11,21 @@ <h5>The latest Alpha MonoDevelop release is: <strong>{{ site.data.alpharelease.v
     <p>Please choose your operating system to view the available packages. Source code is available on <a href="https://github.com/mono/monodevelop">GitHub</a> (viewing only) or as a <a href="https://download.mono-project.com/sources/monodevelop/">Tarball</a>.</p>
 
     <dl id="mono-download" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
-      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;Mac OS X</a></dd>
+      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;macOS</a></dd>
       <dd><a href="#download-lin"><i class="fa fa-linux"></i>&nbsp;Linux</a></dd>
       <dd><a href="#download-win"><i class="fa fa-windows"></i>&nbsp;Windows</a></dd>
     </dl>
 
     <div class="tabs-content" id="mono-download-panel">
       <div class="panel content active" id="download-mac" style="padding: 20px 10px 10px 10px">
-        <h3>Visual Studio for Mac is available as a Mac Disk Image (.dmg)</h3>
+        <h3>MonoDevelop for macOS is available from source only</h3>
+        <p>Please refer to the <a href="/developers/building-monodevelop/">building guide</a> for more information about how to install and configure your MonoDevelop.</p>
+        <hr />
+        <h3>Visual Studio for Mac is available to install</h3>
```

Comment:
should we put this section first instead of the MonoDevelop one? I assume it is what most people want.